core/src/main/resources/reference.conf

```diff
@@ -409,7 +426,7 @@ database {
   #driver = "slick.driver.MySQLDriver$"
   #db {
   #  driver = "com.mysql.jdbc.Driver"
-  #  url = "jdbc:mysql://host/cromwell"
+  #  url = "jdbc:mysql://host/cromwell?rewriteBatchedStatements=true"
```

Comment:
This is an odd default considering the default batch size is `1`. Not saying the default is right or wrong one way or the other, but they're currently not really consistent with each other...